cpp/src/centrality/betweenness_centrality_impl.cuh

```diff
@@ -388,10 +549,71 @@ rmm::device_uvector<weight_t> edge_betweenness_centrality(
   bool const normalized,
   bool const do_expensive_check)
 {
-  CUGRAPH_FAIL("Not Implemented");
-  // Edge betweenness is computed like vertex betweenness, but you accumulate
-  // centrality on each edge.  We need to adapt this to support edge properties
-  // properly.
+  //
+  // Betweenness Centrality algorithm based on the Brandes Algorithm (2001)
+  //
+  if (do_expensive_check) {}
```

Comment:
No need to check `vertices_begin, vertices_end` points to valid vertices?

Reply:
Added in next push.